spec.html

```diff
@@ -27105,24 +27105,24 @@ <h1>Number.isSafeInteger ( _number_ )</h1>
       <emu-clause id="sec-number.max_safe_integer">
         <h1>Number.MAX_SAFE_INTEGER</h1>
         <emu-note>
-          <p>The value of `Number.MAX_SAFE_INTEGER` is the largest integer n such that n and n + 1 are both exactly representable as a Number value.</p>
+          <p>The value of `Number.MAX_SAFE_INTEGER` is the largest integral Number n such that n and n + *1*<sub>𝔽</sub> are both exactly representable as a Number value.</p>
         </emu-note>
-        <p>The value of `Number.MAX_SAFE_INTEGER` is 9,007,199,254,740,991 (2<sup>53</sup> - 1).</p>
+        <p>The value of `Number.MAX_SAFE_INTEGER` is *9007199254740991*<sub>𝔽</sub> (𝔽(2<sup>53</sup> - 1)).</p>
         <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
       </emu-clause>
 
       <emu-clause id="sec-number.max_value">
         <h1>Number.MAX_VALUE</h1>
-        <p>The value of `Number.MAX_VALUE` is the largest positive finite value of the Number type, which is approximately <emu-eqn>1.7976931348623157 &times; 10<sup>308</sup></emu-eqn>.</p>
+        <p>The value of `Number.MAX_VALUE` is the largest positive finite value of the Number type, which is approximately 𝔽(<emu-eqn>1.7976931348623157 &times; 10<sup>308</sup></emu-eqn>).</p>
```

Comment:
It's weird to say approximately and then give a specific float. I'd just leave it as a real here.